### PR TITLE
ggg

### PR DESCRIPTION
**Pull request by ckazu**

### CODE REVIEW

**Comment by takesato** [comment]
sandbox あったんだ

**Comment by devinfra** [comment]
Can one of the admins verify this patch?

**Comment by devinfra** [comment]
Can one of the admins verify this patch?

**Comment by devinfra** [comment]
Can one of the admins verify this patch?

File: README.md [hunk 1]
```diff
@@ -7,3 +7,5 @@ hoge fuga
 ## ccc
 
 ## fff 
+
+ggg
```
Comment (vividmuimui) [inline README.md]: hogehoge

File: README.md [hunk 2]
```diff
@@ -7,3 +7,5 @@ hoge fuga
 ## ccc
 
 ## fff 
+
```
Comment (vividmuimui) [inline README.md]: hogehoge2

File: README.md [hunk 3]
```diff
@@ -7,3 +7,5 @@ hoge fuga
 ## ccc
 
 ## fff
```
Comment (vividmuimui) [inline README.md]: hugahuga

File: README.md [hunk 4]
```diff
@@ -7,3 +7,5 @@ hoge fuga
 ## ccc
```
Comment (vividmuimui) [inline README.md]: hugahuga2

Comment (vividmuimui) [inline README.md]: hugahuga

File: README.md [hunk 5]
```diff
@@ -7,3 +7,5 @@ hoge fuga
...
```
Comment (vividmuimui) [inline README.md]: hugahuga2

Comment (ttanimichi) [inline README.md]: sw

Comment (ttanimichi) [inline README.md]: ddd

Comment (takesato) [inline README.md]: テステス

File: README.md (hunk 2 above)
Comment (takesato) [inline README.md]: スレッド

File: README.md (hunk 5 above)
Comment (ckazu) [inline README.md]: https://github.com/rails/rails/blob/df94b863c2ff8f1bcf12e36ed8fc1419292668e7/railties/test/generators/plugin_generator_test.rb#L270-L284

Comment (ckazu) [inline README.md]: https://github.com/aiming/sandbox/pull/8/files#diff-04c6e90faac2675aa89e2176d2eec7d8L2

Comment (ckazu) [inline README.md]: https://github.com/aiming/sandbox/blob/f3e1406ed62fda49beba2d0dd6cb46e484916cb7/helloworld#L1

**Comment by ttanimichi** [comment]
なんか通知が来てびっくりしたｗ

**Comment by ttanimichi** [comment]
この repo って public だったんですね

**Pull request closed without merging**